go/tools/builders/compile.go

```diff
@@ -184,12 +183,22 @@ func compile(args []string) error {
 	if err := cmd.Wait(); err != nil {
 		return fmt.Errorf("error running compiler: %v", err)
 	}
+	pkgDefFile := filepath.Join(filepath.Dir(*output), pkgDef)
```

Comment:
Same comments as in compilepkg.go about moving this code into a block below the nogo stuff.